phx/results/admin.py

```diff
@@ -5,6 +5,17 @@
 from .models import Event, Performance, Result
 
 
+@admin.action(description="Create Results from selected Events")
+def create_result_from_event(_modeladmin, request, queryset):
+    for event in queryset:
+        Result.objects.create(
+            title=f"{event.name} - {event.location}",
+            event_date=event.date,
+            event=event,
+            author=request.user,
+        )
```

Comment:
Is it possible that someone could create a duplicate result from doing this? 

(Never underestimate what people are capable of!) 

I wonder if this should check that there isn't already a result for this event before creating one.

Reply:
Good point - it's definitely is possible. I can put a check in and maybe update the db schema to make it impossible